src/tools/kvstore_tool.cc

```diff
@@ -38,6 +33,17 @@ StoreTool::StoreTool(const string& type, const string& path, bool need_open_db)
   }
 }
 
+void StoreTool::start_bluestore(const string& path, bool need_open_db)
+{
+    BlueStore* bluestore = new BlueStore(g_ceph_context, path);
+    KeyValueDB *db_ptr;
+    int r = bluestore->start_kv_only(&db_ptr, need_open_db);
+    if (r < 0) {
+    //  exit(1);
```

Comment:
why did you comment this out?

Reply:
A mistake, I was commenting out lines to narrow down where the ICE was and I guesss I forgot to uncomment this one.